scripts/install.nsi

```diff
@@ -157,9 +173,28 @@ Function CleanUninstall
     RMDir "$SMPROGRAMS\FreeScribe"
 FunctionEnd
 
+Function CheckForOldConfig
+    ; Check if the old version exists in AppData
+    IfFileExists "$APPDATA\FreeScribe\settings.txt" 0 End
+        ; Open Dialog to ask user if they want to uninstall the old version
+        MessageBox MB_YESNO|MB_ICONQUESTION "An old configuration file has been detected. Would you like to remove it?" IDYES RemoveOldConfig IDNO End
```

Comment:
Maybe change the wording to include "Highly recommended" or something... Just so the user is aware that if they dont it might cause problems

Reply:
We should probably handle old config files in python as well so we can provide backward compatibility. Ill track in a issue.

Reply:
See convo comments this morning in discord @pemba1s1

Reply:
updated to `An old configuration file has been detected. We recommend removing it to prevent conflict with new versions. Would you like to remove it?`